src/renderer/inc/FontInfoBase.hpp

```diff
@@ -60,10 +60,10 @@ class FontInfoBase
     bool IsDefaultRasterFontNoSize() const;
 
 private:
-    WCHAR _wszFaceName[LF_FACESIZE];
-    LONG _lWeight;
-    BYTE _bFamily;

```

Comment:
isn't `BYTE` a def of `unsigned char`? Could use that instead of the Windowsism and it should still be compatible.